lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -0,0 +1,22 @@
+module ManageIQ
+  module Providers
+    module Workflows
+      class BuiltinMethods
+        def self.email(params = {})
+          options = {
+            :to      => params["To"],
+            :from    => params["From"],
+            :subject => params["Subject"],
+            :cc      => params["Cc"],
+            :bcc     => params["Bcc"],
+            :body    => params["Body"]
+          }
+
+          task_id = GenericMailer.deliver_task(:generic_notification, options)
```

Comment:
NOTE this method doesn't exist yet we only have the "fire-and-forget" method but I wanted something with a miq_task to show waiting for it to finish

Reply:
https://github.com/ManageIQ/manageiq/pull/23017